cargo-scout-lib/src/linter/mod.rs

```diff
@@ -8,41 +8,52 @@ pub trait Linter {
 }
 
 #[derive(Deserialize, PartialEq, Debug)]
-pub struct LintCode {
-    pub code: String,
-    pub explanation: String,
+struct LintCode {
+    code: String,
+    explanation: String,
 }
 
 #[derive(Deserialize, PartialEq, Debug)]
-pub struct LintSpan {
-    pub file_name: String,
+struct LintSpan {
+    file_name: String,
     /// The line where the lint should be reported
     ///
     /// GitHub provides a line_start and a line_end.
     /// We should use the line_start in case of multi-line lints.
     /// (Why?)
-    pub line_start: usize,
+    line_start: usize,
 }
 
 #[derive(Deserialize, PartialEq, Debug, Clone)]
```

Comment:
Since the latest stable, we can mark all pub structs with pub fields [`#[non_exhaustive]`](https://blog.rust-lang.org/2019/12/19/Rust-1.40.0.html#[non_exhaustive]-structs,-enums,-and-variants)

Reply:
hmm that's interesting. In the readme I claimed the MSRV was 1.37 but I suppose we can bump it to 1.40 and do it in a subsequent PR.
It would also allow us to slap non_exhaustive to our error enum, just in case :D